src/util/snake-case-mappers.js

```diff
@@ -0,0 +1,34 @@
+import { isArray, isObject, snakeCase, camelCase, mapKeys } from 'lodash';
```

Comment:
I don't think this should be included here. You can use some dummy wrap / postprocess functions in tests which converts identifiers lower / uppercase to demonstrate the problem.

Reply:
can those be simply moved to tests then?

Reply:
There is no reason, why they should be in at all. Just using dummier implementations that does uppercase / lower case operations for identifiers and results should do the same thing and are simpler.